test/cpp/thread_manager/thread_manager_test.cc

```diff
@@ -91,7 +91,7 @@ grpc::ThreadManager::WorkStatus ThreadManagerTest::PollForWork(void **tag,
   // Simulate "polling for work" by sleeping for sometime
   SleepForMs(kPollingTimeoutMsec);
 
-  *tag = nullptr;
+  *tag = NULL;
```

Comment:
Yang mentioned that my test is not in `grpc` namespace while the `nullptr` definition (in `grpc++/impl/codegen/config.h` is in grpc namespace).  I am hoping this would work.